crates/ruma-api/tests/ui/05-request-only.rs

```diff
@@ -10,14 +10,15 @@ ruma_api! {
         description: "Does something.",
         method: POST, // An `http::Method` constant. No imports required.
         name: "some_endpoint",
-        path: "/_matrix/some/endpoint/:baz",
+        path: "/_matrix/some/endpoint/:foo",
         rate_limited: false,
         authentication: None,
     }
 
     #[derive(PartialEq)] // Make sure attributes work
     request: {
         // With no attribute on the field, it will be put into the body of the request.
+        #[ruma_api(path)]
```

Comment:
This comment no longer makes sense now. I think the better fix would be to remove `:baz` from the path.